calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx

```diff
@@ -0,0 +1,38 @@
+22 Mar 2023
+
+### Improvements
```

Comment:
I thinnk this entire block is v3.15.1

Reply:
Yes, by design.

Reply:
Have you tried to look at the preview of the release notes?